pages/_en-US/twilightmenu/how-to-get-box-art.md

```diff
@@ -5,18 +5,24 @@ section: twilightmenu
 category: other
 title: How to Get Box Art
 description: How to get box art/game covers in TWiLight Menu++
+tabs:
+  - windows: Windows
+    macos_linux: macOS/Linux
```

Comment:
iirc this should be `other` for it to properly auto select, I could add support for substring searching though probably.

Reply:
It might be worthwhile to keep the support for substring searching for a different PR, just to keep everything organized.